content/configuration/output.md

```diff
@@ -258,6 +258,13 @@ Note that `output.libraryTarget` defaults to `var`. This means if only `output.l
 T> Read the [authoring libraries guide](/guides/author-libraries) guide for more information on `output.library` as well as `ouput.libraryTarget`.
 
 
+## `output.libraryExport`
+
+`string` or `string[]` (since webpack 3.0.0)
```

Comment:
Just out of curiosity, is `string[]` supposed to mean "array of strings"? I've seen this used in various other places but didn't realize that was the standard way to document it.